src/virtual-select.js

```diff
@@ -681,12 +685,21 @@ export class VirtualSelect {
         return;
       }
 
+      if (this.markSearchResults) {
+        /** remove previous modifications to the label */
+        d.label = d.label.replace(/<\/*mark>/g, '');
+      }
+
       let value = d.label.toString().toLowerCase();
       let isVisible = value.indexOf(searchValue) !== -1;
       d.isVisible = isVisible;
 
       if (isVisible) {
         visibleOptionsCount++;
+        if (this.markSearchResults) {
```

Comment:
This block could be changed as below. This way we can have multiple `mark` tag in a lable.

```js
if (markSearchResults) {
  d.label = d.label.replace(new RegExp(`(${searchValue})`, 'gi'), `<mark>$1</mark>`);
}
```